src/communication/azext_communication/manual/_params.py

```diff
@@ -8,17 +8,147 @@ def load_arguments(self, _):
+
+    with self.argument_context('communication chat add-participant') as c:
+        c.argument('thread_id', options_list=['--thread-id', '-t'],
+                   type=str, help='Thread id')
+        c.argument('user_id', options_list=['--user-id', '-u'],
+                   type=str, help='Chat participant identifier')
+        c.argument('display_name', options_list=['--display-name', '-d'],
+                   type=str, help='Chat participant display name')
+        c.argument('start_time', options_list=['--start-time', '-s'],
+                   type=str, help='Start time to share history in ISO8601 format, ex: 2022-07-14T10:21')
+
+    with self.argument_context('communication chat remove-participant') as c:
+        c.argument('thread_id', options_list=['--thread-id', '-t'],
+                   type=str, help='Thread id')
+        c.argument('user_id', options_list=['--user-id', '-u'],
+                   type=str, help='Chat participant identifier')
+
+
+def _load_chat_message_management(self):
+    with self.argument_context('communication chat send-message') as c:
+        c.argument('thread_id', options_list=['--thread-id', '-t'],
+                   type=str, help='Thread id')
+        c.argument('display_name', options_list=['--display-name', '-d'],
+                   type=str, help='Sender''s display name')
+        c.argument('message_content', options_list=['--content', '-c'],
+                   type=str, help='Chat message content')
+        c.argument('message_type', options_list=['--message-type', '-y'],
+                   type=str, help='Content type, can be text or html')
+
+    with self.argument_context('communication chat list-messages') as c:
+        c.argument('thread_id', options_list=['--thread-id', '-t'],
+                   type=str, help='Thread id')
+        c.argument('start_time', options_list=['--start-time', '-s'],
+                   type=str, help='Start time in ISO8601 format, ex: 2022-07-14T10:21')
+
+    with self.argument_context('communication chat get-message') as c:
+        c.argument('thread_id', options_list=['--thread-id', '-t'],
+                   type=str, help='Thread id')
+        c.argument('message_id', options_list=['--message-id', '-i'],
+                   type=str, help='Message id')
+
+    with self.argument_context('communication chat update-message') as c:
+        c.argument('thread_id', options_list=['--thread-id', '-t'],
+                   type=str, help='Thread id')
+        c.argument('message_id', options_list=['--message-id', '-i'],
+                   type=str, help='Message id')
+        c.argument('message_content', options_list=['--content', '-c'],
+                   type=str, help='Chat message content')
+
+    with self.argument_context('communication chat delete-message') as c:
+        c.argument('thread_id', options_list=['--thread-id', '-t'],
+                   type=str, help='Thread id')
+        c.argument('message_id', options_list=['--message-id', '-i'],
```

Comment:
Why you always add `-*` for all arguments? It's not recommended. Those short argument alias may conflicted with the global arguments. Please add them only when needed.

Reply:
Removed short arguments for most commands.